### DISCUSSION TITLE
Flows Improvement

### DISCUSSION THREAD

**Discussion by NilsBaumgartner1994** [directus/directus#20950] [Feature Requests]
### Summary

So flows are nice and i like them but they lack of certain functions.
- Import/Export: Since you can work in the web its nice. But the content is stored in the database. The api supports the import/export of these but some UI buttons would be nice.
- Copy Flow: Please let us copy an existing flow
- Extract into new flow: After creating a quite big flow it would be nice so seperate it into smaller chunks which could be reused in other flows.

### Motivation

-

### Detailed Design

-

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

-

### Alternatives

-

### Adoption Strategy

-

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
